rpm.mk

```diff
@@ -47,31 +47,32 @@ bundle-rust:
 
 install-node-modules:
 ifeq ($(COCKPIT_ON), 1)
-	cd src/cockpit/389-console; make -f node_modules.mk install
+	cd src/cockpit/389-console; \
+	rm -rf node_modules; \
+	npm install > /dev/null;
 endif
 
 build-cockpit: install-node-modules
 ifeq ($(COCKPIT_ON), 1)
 	cd src/cockpit/389-console; \
 	rm -rf cockpit_dist; \
-	NODE_ENV=production make -f node_modules.mk build-cockpit-plugin
+	NODE_ENV=production ./build.js; \
+	cp -r dist cockpit_dist
 endif
 
 dist-bz2: install-node-modules download-cargo-dependencies
 ifeq ($(COCKPIT_ON), 1)
 	cd src/cockpit/389-console; \
 	rm -rf cockpit_dist; \
-	NODE_ENV=production make -f node_modules.mk build-cockpit-plugin; \
-	mv node_modules node_modules.release; \
```

Comment:
These lines were for the case that we don't need all of the npm packages in our release tarball.

And I think, Cockpit has the same idea:
https://github.com/cockpit-project/cockpit/blob/main/package.json#LL3C61-L3C61

For some reason, they still have something in their tarball but it's minimal:
~/src/cockpit/cockpit-294.1/node_modules on rawhide ?1 ❯ ll                                                                                                                     at 09:02:39 AM
total 0
drwxr-xr-x. 1 spichugi spichugi 170 Jun 15 04:09 chrome-remote-interface
drwxr-xr-x. 1 spichugi spichugi  92 Jun 15 04:09 commander
drwxr-xr-x. 1 spichugi spichugi  94 Jun 15 04:09 sizzle
drwxr-xr-x. 1 spichugi spichugi  98 Jun 15 04:09 ws

Reply:
These lines also caused build failures because the following "tar" command expects node_modules to exist.

Reply:
For the tar command (and eventually for the build time later on) we had `touch -r src/cockpit/389-console/package.json $(NODE_MODULES_TEST)` which essentially created an empty file that's sole purpose was to "trick" npm into desired usage.

Reply:
But there is no node_modules directory, so the tar command fails

Reply:
Okay, my bad: to be presize, - previously, this helped `tar` command to succeed (and be correctly "tricked" into the desired state as I previously mentioned):

```
NODE_MODULES_TEST = src/cockpit/389-console/node_modules/webpack
mkdir -p $(NODE_MODULES_TEST)
touch -r src/cockpit/389-console/package.json $(NODE_MODULES_TEST)
```